### ISSUE TITLE
 No dashboards are active for the current data set. 

### ISSUE THREAD

**Issue by Nisargpatel16** [zzyunzhi/object-intrinsics#2]
After running the command "torchrun --standalone --nnodes=1 --nproc_per_node=1 scripts/train.py -d data/example" I am getting the below attached error on port 6006. Please help me with the same.
![Error](https://github.com/zzyunzhi/object-intrinsics/assets/72245360/f3039c51-9d76-46ba-ad6c-2928b7267592)
![error2](https://github.com/zzyunzhi/object-intrinsics/assets/72245360/9e58c9f0-1f2c-45b0-bad0-0f7144ab9e43)

**Comment by zzyunzhi** [comment]
Could you check if `logs/_data_example_train` contains tensorboard event files? If so, it's possible that you need to wait a little bit for tensorboard to be loaded.

**Comment by Nisargpatel16** [comment]
Yes it contains event file. Also I have waited for more than 12 hours but still it is not getting updated.
Also I ran the command "tensorboard --inspect --logdir DIRPATH" which gives below output.

Found event files in:
-- $logdir

Event Statistics : 
![Error3](https://github.com/zzyunzhi/object-intrinsics/assets/72245360/6b1155fc-a87a-4f11-ae34-c11ae220148f)

Tags : 
![Error4](https://github.com/zzyunzhi/object-intrinsics/assets/72245360/d4385719-87fb-41d4-ba41-94ce47eea1ad)

**Comment by zzyunzhi** [comment]
Could you check the console output to make sure that training is not halted somewhere before the tensorboard is written (which occurs here https://github.com/zzyunzhi/object-intrinsics/blob/24d6016079c9c8f49128fa80c409bf8153cbe8aa/src/trainers/gan_pose_trainer.py#L310)?

**Comment by Nisargpatel16** [comment]
Checked it, the observation was as followed : -
- Actually the program does not go inside the for loop.
https://github.com/zzyunzhi/object-intrinsics/blob/24d6016079c9c8f49128fa80c409bf8153cbe8aa/src/trainers/gan_pose_trainer.py#L286
- Also the console output is attached herewith FYI.
![error3](https://github.com/zzyunzhi/object-intrinsics/assets/72245360/b400fd4c-1832-4155-8c23-759efcabc83c)
